medicines/search/src/azure_rest.rs

```diff
@@ -4,6 +4,8 @@ pub async fn make_post_request_with_body(
     api_key: &str,
 ) -> Result<(), reqwest::Error> {
     let client = reqwest::Client::new();
+    println!("{}", url);
```

Comment:
is this for debugging, and inadvertently left in?

Reply:
Yes, yes it is :P Doh! Just removed.